tests/phpunit/CRM/Export/BAO/ExportTest.php

```diff
@@ -2183,32 +2184,33 @@ protected function getBasicHeaderDefinition($isContactExport) {
       59 => 'Postal Code',
       60 => 'Latitude',
       61 => 'Longitude',
-      62 => 'Address Name',
-      63 => 'Master Address Belongs To',
-      64 => 'County',
-      65 => 'State',
-      66 => 'Country',
-      67 => 'Phone',
-      68 => 'Phone Extension',
-      69 => 'Phone Type',
-      70 => 'Email',
-      71 => 'On Hold',
-      72 => 'Use for Bulk Mail',
-      73 => 'Signature Text',
-      74 => 'Signature Html',
-      75 => 'IM Provider',
-      76 => 'IM Screen Name',
-      77 => 'OpenID',
-      78 => 'World Region',
-      79 => 'Website',
-      80 => 'Group(s)',
-      81 => 'Tag(s)',
-      82 => 'Note(s)',
+      62 => 'Is manually geocoded',
```

Comment:
@seamuslee001 this looks  out of place due to the capitalisation - shall we change it in the schema? The rest  use title case

Reply:
Done yeh i think that is probably sensible